### DISCUSSION TITLE
Using ginkgo tests for go kata

### DISCUSSION THREAD

**Discussion by psaikido** [codewars/codewars.com#3259] [Q&A]
How to run a test locally on a linux computer? I've tried a load of different things but can't get the tests working. My code works in the browser when I click 'test'. I installed ginkgo with yay. When I download the main.go and the test.go to a local directory and run 'ginkgo' I get "found no test suites". If I run 'go test' I get a message that it has found the packages 'kata' and 'kata_test' but it doesn't run the actual test. How should this be done? I'm starting with a really simple one so that I'm set up for the more difficult ones later. The test in question is https://www.codewars.com/kata/5ce9c1000bab0b001134f5af

**Comment by psaikido** [comment]
The version of go being used in these kata, v1.20, is out of date. I think the problems I'm facing in trying to get any test to run on a local linux machine comes from this. I'm seeing lots of versioning and dependency mismatch problems with go and ginkgo-cli. Please update and provide some instructions on how to get started.

**Reply by hobovsky** [reply]
Update of Go setup has already been requeated: 
 https://github.com/codewars/runner/issues/257

**Comment by psaikido** [comment]
I found a workaround to use 'go test' and not use ginkgo at all here - https://stackoverflow.com/questions/44782807/ginkgo-tests-not-being-found
